content/applications/inventory_and_mrp/inventory/product_management/product_tracking/lots.rst

```diff
@@ -1,201 +1,171 @@
-      :alt: List of lot numbers copied on excel spreadsheet.
+After clicking away from the text field, Odoo automatically generates the necessary lot number
+lines. The quantities in the :guilabel:`Done` column match the first entry. Manually adjust the
+:guilabel:`To` locations and :guilabel:`Done` quantities for each lot number, as needed.
 
 Once all product quantities have been assigned a lot number, click :guilabel:`Confirm` to close the
-pop-up. Then, click :guilabel:`Validate`.
-
-A :guilabel:`Traceability` smart button appears upon validating the receipt. Click the
-:guilabel:`Traceability` smart button to see the updated :guilabel:`Traceability Report`, which
-includes: a :guilabel:`Reference` document, the :guilabel:`Product` being traced, the
-:guilabel:`Lot/Serial #` assigned, and more.
-
-Manage lots on delivery orders
-------------------------------
+pop-up. Then, click :guilabel:`Validate` on the receipt form.
 
-Assigning lot numbers to **outgoing** goods can be done directly from the sales order (SO).
+.. seealso::
+   :ref:`Traceability report for lot numbers <inventory/product_management/lot-traceability>`
 
-To create an :abbr:`SO (sales order)`, go to the :menuselection:`Sales app --> Create`. Doing so
-reveals a new, blank quotation form.
+.. _inventory/product_management/assign-lots-delivery:
 
-On this blank quotation form, fill out the necessary information by adding a :guilabel:`Customer`,
-and adding products to the :guilabel:`Product` lines (in the :guilabel:`Order Lines` tab) by
-clicking :guilabel:`Add a product`.
+On delivery orders
+------------------
 
-Then, choose the desired quantity to sell by changing the number in the :guilabel:`Quantity` column.
+Specify which lot numbers for a product are chosen for outgoing shipment on the delivery order form.
 
-Once the quotation has been filled out, click the :guilabel:`Confirm` button to confirm the
-quotation. When the quotation is confirmed, it becomes an :abbr:`SO (sales order)`, and a
-:guilabel:`Delivery` smart button appears.
+To begin, create or select an existing |SO| from the :menuselection:`Sales` app. After confirming
+the |SO|, the :guilabel:`Delivery` smart button becomes available. Click the :guilabel:`Delivery`
+smart button to view the warehouse receipt form for that specific |SO|.
 
-Click the :guilabel:`Delivery` smart button to view the warehouse receipt form for that specific
-:abbr:`SO (sales order)`.
+.. note::
+   Alternatively, navigate to delivery orders by going to the :menuselection:`Inventory` app and
+   clicking the :guilabel:`Delivery Orders` Kanban card.
 
-From here, click the :guilabel:`Additional Options` menu, represented by a `hamburger` icon (four
-horizontal lines, located to the right of the :guilabel:`Unit of Measure` column in the
-:guilabel:`Operations` tab). Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
+On the delivery order form, in the :guilabel:`Detailed Operations` tab, click the :guilabel:`⦙≣
```

Comment:
since this isn't part of the note admonition, the reader, at this point, is on the WH receipt, via the Delivery smart button from the SO form. To get to a delivery order form, should they navigate to the 'Delivery Orders' kanban card, as described in the admonition? Some clarification here could be helpful. Thanks

Reply:
Also, I'm not seeing a 'Detailed Operations' tab...just an 'Operations' tab on the Delivery Orders present on the v16 runbot that i'm using for this review...

Reply:
UPDATE: I think you simply meant to put 'Operations' tab -- just make sure to double-check this during your revisions. thanks!

Reply:
Addressed your comment by adjusting this section to say:
> Clicking the :guilabel:`Delivery` smart button opens the the delivery order form, where lot numbers
are picked for delivery. In the :guilabel:`Operations` tab, click the :guilabel:`⦙≣ (bulleted list)`
icon to the right of the product that is tracked by lot numbers. Clicking that icon reveals a
:guilabel:`Detailed Operations` pop-up window.